.github/workflows/sandbox-build-push.yml

```diff
@@ -56,6 +56,18 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
+      - name: Get image tag
+        id: tag-image
+        run: |
+          if [[ ! -z "${{ needs.set_dev_tags.outputs.image_tag }}" ]]; then \
```

Comment:
Is there a way this can be commented without breaking the embedding? Or commented at the yaml level before the nested bash script?

My interpretation of the logic is (and I'm pretty confident I'm getting something wrong...):
```
if requested_dev_tag is not empty:
  tag_to_use = dev_tag
if requested_release_tag is not empty:
  if tag_to_use is not empty:  # Ie if a dev tag WAS already populated
    tag_to_use = tag_to_use,requested_release_tag  # Overwrite the dev tag? Or return a pair of tags, comma separated?
  else:  # there was no dev tag
    tag_to_use = requested_release_tag  # return just the release tag
```